### ISSUE TITLE
Update ERG Demographics

### ISSUE THREAD

**Issue by Dbarnes1** [Esri/solutions-erg-widget#20]
Currently, we seem to be using a set of broken demographic layers in our EmergencyOperations map service to drive the demographic layers. It seems like SampleServer6 has a functional map service for demographic data, although it may still be outdated. 

These are our layers in the EmergencyOperations Map Service, they don't seem to be up to date/working: 
![image](https://user-images.githubusercontent.com/13024357/31198505-609695dc-a909-11e7-9012-231f1c64bc36.png)

What we should do is update the widget so that it pulls in a service from ArcGIS Online. We may want to explore data being released as part of Community Analyst. We could include a sample service, but we want users to be able to go to the configuration page of the widget and pull in whatever demographic service they want.

**Comment by Dbarnes1** [comment]
not including demographic analysis at this release, moving to version after next.

**Comment by adgiles** [comment]
I don't think this issue is valid anymore with how we have designed the widget. The widget should be used in conjunction with the situation awareness widget and use the power of the feature actions that this widget provides. So the workflow would be similar to:

Create ERG Zones:

**Comment by topowright** [comment]
This is a good example of how we could build a Solutions around this widget or this workflow using the widget.

**Comment by topowright** [comment]
We should work with @NikkiGolding and have a good State and EM approach to this issue. Having her input on this issue is very much needed to make sure we do not duplicate efforts.
